components/brave_rewards/resources/internals/components/log.tsx

```diff
@@ -110,7 +110,7 @@ export class Log extends React.Component<Props, State> {
         </LogControls>
         <LogTextArea value={this.props.log} readOnly={true} onContextMenu={this.preventDefault}/>
         <Notice>
-          {getLocale('logNotice', { numberOfLines: '5,000' })}
+          {getLocale('logNotice', { numberOfLines: '20,000' })}
```

Comment:
Just updating this to reflect reality.

Reply:
20,000 lines on mobile might cause performance issues, so in the past we set to 5,000 for the UI and 20,000 for the file

Reply:
Fixed.